clippy_lints/Cargo.toml

```diff
@@ -33,8 +31,6 @@ walkdir = "2.3"
 
 [features]
 deny-warnings = ["clippy_config/deny-warnings", "clippy_utils/deny-warnings"]
```

Comment:
Should the `clippy_deprecated_lints` also get the `deny_warnings` feature? I guess it is not really necessary, as that file/crate is fully generated?

Reply:
It shouldn't have anything in it to warn about given how little is there.